docs/labs/JavavsCPP.md

```diff
@@ -22,6 +22,23 @@ I would say that they fit these pictures pretty well
 | ![http://www.pxleyes.com/images/users/S/Sarah06/3485/fullsize/4e5f610b59f3a.jpg](diffjavacpp/media/image1.jpeg) | ![http://www.cityofws.org/portals/0/images/police/storytime2\_233.jpg](diffjavacpp/media/image2.jpeg) |
 Source: left - *pxleyes.com*, right - *[cityofws](http://www.cityofws.org)*
 
```

Comment:
I think we should create a heading and some explanatory sentence before showing the table so folks know that they are pretty much the same.